force-app/main/default/classes/RelationshipLookupMapper_TEST.cls

```diff
@@ -25,68 +25,273 @@
-            RelationshipLookupMapper relationshipLookupMapperInstance2 = RelationshipLookupMapper.getInstance();
+        RelationshipLookupMapper relationshipLookupMapperInstance1 = RelationshipLookupMapper.getInstance();
+        RelationshipLookupMapper relationshipLookupMapperInstance2 = RelationshipLookupMapper.getInstance();
         Test.stopTest();
 
-        System.assertEquals(RelationshipLookupMapper.instance, relationshipLookupMapperInstance1, 'Instance of mapper class returned should match static instance variable.');
-        System.assertEquals(relationshipLookupMapperInstance1, relationshipLookupMapperInstance2, 'Subsequent retrievals of mapper class instance should return existing instance.');
-        System.assertEquals(true, relationshipLookupMapperInstance1 != null, 'Instance of mapper class should not be null.');
+        System.assertEquals(
+            RelationshipLookupMapper.instance,
+            relationshipLookupMapperInstance1,
+            'Instance of mapper class returned should match static instance variable.'
+        );
+        System.assertEquals(
+            relationshipLookupMapperInstance1,
+            relationshipLookupMapperInstance2,
+            'Subsequent retrievals of mapper class instance should return existing instance.'
+        );
+        System.assertEquals(
+            true,
+            relationshipLookupMapperInstance1 != null,
+            'Instance of mapper class should not be null.'
+        );
     }
 
     /****************************************************************************************************************************
-    * @description Test method to verify that the getRelationshipLookupModels method returns a list of relationship lookup models
-    *****************************************************************************************************************************/
+     * @description Test method to verify that the getRelationshipLookupModels method returns a list of relationship lookup models
+     *****************************************************************************************************************************/
     @isTest
     private static void getRelationshipLookupModels() {
-
-        insert new Relationship_Lookup__c(Name = 'test' + system.now(), Active__c = true, Female__c = 'wife',
-                                          Male__c = 'husband', Neutral__c = 'spouse');
+        insert new Relationship_Lookup__c(
+            Name = 'test' + system.now(),
+            Active__c = true,
+            Female__c = 'wife',
+            Male__c = 'husband',
+            Neutral__c = 'spouse'
+        );
 
         Test.startTest();
-             List<RelationshipLookupModel> relationshipLookupModelsList = RelationshipLookupMapper.getInstance().getRelationshipLookupModels();
+        List<RelationshipLookupModel> relationshipLookupModelsList = RelationshipLookupMapper.getInstance()
```

Comment:
This is not a required change and something I've been commenting very loosely upon. Consider what you are testing uniquely within the test block and what metrics you are looking to retrieve? Is getting the instance part of those metrics or are we testing the performance of an individual method?

Reply:
I think this is just a formatting change in this specific place